config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  #users
+  root "users#index"
+  get "/users/:id", to: "users#show"
+  #posts
+  get "users/:id/posts", to: "posts#index"
+  get "users/:id/posts/:id", to: "posts#show"
```

Comment:
- [ ] Your routes are not showing index views of the users and posts. It would be nice to have controller index and show methods so the routes point to them from here. You could implement the routes like this:

```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

It's ok to replace the highlighted code with this one  👍🏽

Reply:
well resolved! ✔️